src/components/Header.tsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import { SearchBar } from './SearchBar';
 import styles from '../styles/App.module.css'
 
-interface HeaderProp{
+interface HeaderProp {
   onSearch: (searchTerm: string) => void;
+  showSearch: boolean; 
 }
 
-export const Header: React.FC<HeaderProp> = ({ onSearch }) => {
+export const Header: React.FC<HeaderProp> = ({ onSearch, showSearch }) => {
```

Comment:
Nit: Can provide a default value to the prop.

```suggestion
export const Header: React.FC<HeaderProp> = ({ onSearch, showSearch = false }) => {
```

Reply:
fixed https://github.com/Sakshi-Simform/Recipe-App/pull/4/commits/1ece3104a4c26ac63419445ab6c1a2546845fe90